packages/flutter_tools/lib/src/ios/device_ios.dart

```diff
@@ -331,9 +324,12 @@ class IOSSimulator extends Device {
 
   static List<IOSSimulator> getAttachedDevices([IOSSimulator mockIOS]) {
     List<IOSSimulator> devices = [];
-    _IOSSimulatorInfo deviceInfo = _getRunningSimulatorInfo(mockIOS);
-    if (deviceInfo != null)
-      devices.add(new IOSSimulator(id: deviceInfo.id, name: deviceInfo.name));
+    try {
+      _IOSSimulatorInfo deviceInfo = _getRunningSimulatorInfo(mockIOS);
+      if (deviceInfo != null)
+        devices.add(new IOSSimulator(id: deviceInfo.id, name: deviceInfo.name));
+    } catch (e) {
+    }
```

Comment:
Lots of other places in this file silently ignore exceptions. :/  I'm going to leave this as-is for now.